includes/class-sensei-modules.php

```diff
@@ -1054,21 +1058,24 @@ public function calculate_user_module_progress( $user_id = 0, $module_id = 0, $c
 	}
 
 	/**
-	 * Register admin screen for ordering modules
+	 * Register admin pages related to modules.
 	 *
 	 * @since 1.8.0
-	 * @deprecated 4.0.0
 	 *
 	 * @return void
 	 */
 	public function register_modules_admin_menu_items() {
-		_deprecated_function( __METHOD__, '4.0.0' );
 
-		// add the modules link under the Course main menu
-		add_submenu_page( 'edit.php?post_type=course', __( 'Modules', 'sensei-lms' ), __( 'Modules', 'sensei-lms' ), 'manage_categories', 'edit-tags.php?taxonomy=module', '' );
+		// Register an admin page for module ordering.
+		add_submenu_page(
+			null, // Hide the submenu.
```

Comment:
We're using `options.php` in the [setup wizard](https://github.com/Automattic/sensei/blob/master/includes/admin/class-sensei-setup-wizard.php#L116) because I believe `null` caused some issue. Should we do the same here?

Reply:
Sure, updated in 7595326c30bcee86e450621f695202e6c4a12371.

Just a heads up - with `options.php` I wasn't able to change the active menu to `Sensei LMS -> Modules` which will be needed in a future PR.

I've tried using @merkushin's method from [here](https://github.com/Automattic/sensei/pull/4672/files#diff-4a68534038d10e99b32ebd25a65b40e4b44bcdb19547b535c23b11b0d7057cf7R164), but it didn't work. This method only works when using `null`.

Reply:
OK. I think it caused some problem with the title for the setup wizard when using `null`, but if we're not noticing any issues then we don't have to do it the same way, especially if it makes highlighting the active menu problematic.

Reply:
I've tested it with WordPress 5.6 and it works fine, so I've decided to revert the change.